src/contracts/libraries/SlashingLib.sol

```diff
@@ -150,10 +151,11 @@ library SlashingLib {
 
         // Step 3: Calculate newStakerDepositScalingFactor
         /// forgefmt: disable-next-item
-        uint256 newStakerDepositScalingFactor = newShares
+        uint184 newStakerDepositScalingFactor = newShares
```

Comment:
Why change this to uint184?

Reply:
Changed type to pack the struct into a single slot, we could use a smaller type if it makes sense.

Reply:
may be a better idea to divide the extra bits to deposit scaler and beacon chain scaler